src/main/java/Todo.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Represents a todo task to be recorded. A <code>Todo</code> object
+ * is represented by one String
+ * e.g., <code>borrow book</code>
+ */
+public class Todo extends Task {
+
+    public Todo(String description) {
+        super(description);
+    }
+
+    public Todo(String description, boolean isDone) {
+        super(description, isDone);
+    }
+
+    /**
+     * Returns summary of task
+     *
+     * @return task summary
+     */
+    @Override
+    public String summary() {
```

Comment:
nit: for method names try to use verbs instead (from textbook), maybe something like getSummary() will be better